tests/parametric/test_headers_precedence.py

```diff
@@ -710,6 +710,7 @@ def _test_headers_precedence_propagationstyle_includes_tracecontext_correctly_pr
                     ["x-datadog-tags", "_dd.p.tid=1111111111111111"],
                 ],
             )
+            print(f"zach here is headers1: {headers1}")
```

Comment:
Forgotten traces? You can use the logger:
`from utils.tools import logger"`

Reply:
:face-palm: 
Yeah, I tried using the logger first IIRC and wasn't able to see the output. Anyways, will delete and thank you for catching.